jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/SeleniumTestHelper.java

```diff
@@ -22,6 +22,8 @@
 
     private static final String SELENIUM_PROPERTY_PATH = "selenium.property.path";
     private static final String TEST_PROPERTIES = "selenium/test.properties";
+    private static final String JWALA_PROPERTIES = "selenium/jwala.properties";
```

Comment:
Please commit a sample property file just like test.properties so that it is easier for developers to digest the purpose and contents of the file